packages/message/src/Granules.ts

```diff
@@ -94,15 +94,16 @@ export const getGranuleQueryFields = (
  * Calculate granule product volume, which is the sum of the file
  * sizes in bytes
  *
- * @param {Array<Object>} granuleFiles - array of granule files
- * @returns {Integer} - sum of granule file sizes in bytes
+ * @param {Array<Object>} granuleFiles - array of granule file objects that conform to the
+ *                                       Cumulus 'api' schema
+ * @returns {string} - sum of granule file sizes in bytes as a string
  */
-export const getGranuleProductVolume = (granuleFiles: ApiFile[] = []): number => {
-  if (granuleFiles.length === 0) return 0;
-  return granuleFiles
+export const getGranuleProductVolume = (granuleFiles: ApiFile[] = []): string => {
```

Comment:
This actually fails to address the concern noted in the ticket, as we need to cast .size to BigInt to do the math, then cast to sting.  Will sort shortly, and add a unit to address sizes larger than Number.MAX_SAFE_INTEGER.

Reply:
Sorted: 879007d10